python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -51,6 +62,20 @@ def destination_id(self) -> str:
     def is_connected(self) -> bool:
         return self.setup_state == FivetranConnectorSetupStateType.CONNECTED.value
 
+    @property
+    def is_paused(self) -> bool:
+        return self.paused
+
+    def assert_syncable(self) -> bool:
```

Comment:
nit: maybe validate_syncable? Just because assert is a python keyword, and we aren't using it here

Reply:
Done in [8e3c5a9](https://github.com/dagster-io/dagster/pull/25911/commits/8e3c5a97fcce6fa46e3976e73b0774a09c7c6e84)